README.md

```diff
@@ -1,14 +1,19 @@
-# Default starter for Cherry On Tech's Gridsome website
+# Welcome to Cherry On Tech's website
 
-This is the project you get when you run `gridsome create new-project`.
+We are a tech squad! We support each other as we learn and grow in our tech careers. We got the idea from Mo Hampton's JuneteenthConf talk, "Breaking into Tech." Learn more about us in an upcoming blog post!
 
-### 1. Install Gridsome CLI tool if you don't have
+## Tech stack
 
-`npm install --global @gridsome/cli`
+This website is built using Gridsome, a static site generator based on VueJS. It's under pretty heavy construction, but we love it already, and we hope you do, too!
 
-### 2. Create a Gridsome project
+## Installation
 
-1. `gridsome create my-gridsome-site` to install default starter
-2. `cd my-gridsome-site` to open the folder
-3. `gridsome develop` to start a local dev server at `http://localhost:8080`
-4. Happy coding 🎉🙌
+1. Install Gridsome CLI tool if you don't have it: `npm install --global @gridsome/cli` or `yarn global add @gridsome/cli`
+1. Clone the repo: `git clone https://github.com/cherryontech/website.git`
```

Comment:
do you know why the numbered list shows up as all 1.'s here but in the rendered markdown its the correct list? weird

Reply:
I'm not sure how exactly it compiles down into the proper numbers, but it makes it easier so when you're writing the markdown file you can rearrange list items.